www/src/examples/Accordion/AllCollapse.js

```diff
@@ -1,10 +1,7 @@
 <Accordion>
   <Card>
     <Card.Header>
-      <Accordion.Toggle
-        onClick={key => console.log(`I've been clicked by : ${key}`)}
-        eventKey="0"
-      >
+      <Accordion.Toggle as={Button} variant="link" eventKey="0">
```

Comment:
so, the default of `<button>` here is a little weird in retrospect. the TWBS example upstream uses `<Button variant="link">`, as this makes things work, but i wonder if we should just use `<SafeAnchor>` instead